scripts/format_code.py

```diff
@@ -70,7 +69,7 @@ def does_file_need_formatting(filename):
     an identical file.
   """
   args = ['clang-format', '-style=file', '-output-replacements-xml', filename]
-  result = subprocess.run(args, stdout=subprocess.PIPE)
+  result = subprocess.run(args, stdout=subprocess.PIPE, check=True)
```

Comment:
It looks like `result` is unused. Can it be removed? Also on line 89.

Reply:
It's in the following line:
```
 for line in result.stdout.decode('utf-8').splitlines():
 ```